src/css/base/_body.scss

```diff
@@ -6,7 +6,7 @@
 // Styleguide Base.body
 body {
 	background-color: $color-sand; // Extends background color to full width on an iPhone X in landscape mode: http://stephenradford.me/removing-the-white-bars-in-safari-on-iphone-x/
-	font-size: 1em;
+	font-size: $font-size-body;
```

Comment:
Can we revert this to `1em`? We're purposefully using an `em` unit on the body to better interface with any custom font sizing the user may have done.